webservices/resources/reports.py

```diff
@@ -73,6 +73,12 @@ def get_range_filters():
     ]
     return filter_range_fields
 
+def get_match_filters():
+    filter_match = [
+        ('filer_type', models.CommitteeReports.means_filed)
+    ]
+    return filter_match
```

Comment:
I can take a look at this again.  Experience has taught me that time away from problems helps solve them.